src/Nethermind/Nethermind.Core.Test/Modules/TestMergeModule.cs

```diff
@@ -30,11 +30,6 @@ protected override void Load(ContainerBuilder builder)
             .AddModule(new MergePluginModule())
 
             .AddSingleton<IBlockFinalizationManager, ManualBlockFinalizationManager>()
-            .OnActivate<IMainProcessingContext>(((context, componentContext) =>
```

Comment:
Moved to `BaseMergePluginModule`. This now work outside of test since `IMainProcessingContext` is DI.